qemu/tests/single_driver_install.py

```diff
@@ -138,7 +138,8 @@ def run(test, params, env):
 
         if params.get_boolean('need_destroy'):
             vm.destroy()
-            vm.create(params=params)
+            params = params.object_params(vm.name)
```

Comment:
This line can be remove.  What i mean is the following 2 lines 
```
params = params.object_params(vm.name)
vm.create(params=params)
```
can be replaced with `vm.create()`

Reply:
@yanan-fu  Got it  Thanks, will check it on framework  to see how it works.